buildSrc/src/main/java/org/elasticsearch/gradle/precommit/PrecommitTask.java

```diff
@@ -36,7 +36,6 @@ public File getSuccessMarker() {
 
     @TaskAction
     public void writeMarker() throws IOException {
-        getSuccessMarker().getParentFile().mkdirs();
```

Comment:
Why is this removed? Don't we need to ensure the parent dir exists?

Reply:
Good question. This is actually unnecessary because annotating `getSuccessMarker()` with `@OutputFile` already ensures that file and any parent directories will be created for us.